File: Bungii.Test.Integration/src/test/java/com/bungii/android/pages/customer/BungiiCompletePage.java
package com.bungii.android.pages.customer;

import com.bungii.common.core.PageBase;
import org.openqa.selenium.WebElement;

public class BungiiCompletePage extends PageBase {


    public WebElement PageTitle_BungiiComplete() { return findElement("//android.widget.TextView[@text='BUNGII COMPLETE']",LocatorType.XPath); }

    public WebElement CloseRateTipPage() { return findElement("//android.view.View[@id='com.bungii.customer:id/toolbar']/android.widget.ImageView",LocatorType.XPath); }

    public WebElement Title_RateYourDriver() { return findElement("//android.widget.TextView[@text ='Rate Your driver']",LocatorType.XPath); }

    public WebElement Image_Driver() { return findElement("com.bungii.customer:id/driver_profile_image",LocatorType.Id); }

    public WebElement DriverName() { return findElement("//android.widget.TextView[@instance='2']",LocatorType.XPath); }

    public WebElement RatingBar() { return findElement("com.bungii.customer:id/rating_bar",LocatorType.Id); }

    public WebElement Title_Tip() { return findElement("//android.widget.TextView[@text='Tip']",LocatorType.XPath); }

    public WebElement TipAmount() { return findElement("com.bungii.customer:id/tip_value_label",LocatorType.Id); }

    public WebElement Button_IncreaseTip() { return findElement("com.bungii.customer:id/tip_increase_button",LocatorType.Id); }

    public WebElement Button_DecreaseTip() { return findElement("com.bungii.customer:id/tip_decrease_button",LocatorType.Id); }

    public WebElement BungiiTime() { return findElement("com.bungii.customer:id/summary_value_time",LocatorType.Id); }

    public WebElement Distance() { return findElement("com.bungii.customer:id/summary_value_distance",LocatorType.Id); }

    public WebElement FinalCost() { return findElement("com.bungii.customer:id/summary_value_distance",LocatorType.Id);}

    public WebElement Button_OK() { return findElement("com.bungii.customer:id/pickup_success_ok_button",LocatorType.Id); }
}

File: BungiiAutomation/Bungii.Test.Regression.Android.Integration/Pages/Bungii/BungiiCompletePage.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Support.PageObjects;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bungii.Test.Regression.Android.Integration.Pages.Bungii
{
    class BungiiCompletePage
    {
        public BungiiCompletePage(IWebDriver driver)
        {
            PageFactory.InitElements(driver, this);
        }

        [FindsBy(How = How.XPath, Using = "//android.widget.TextView[@text='BUNGII COMPLETE']")]
        public IWebElement PageTitle_BungiiComplete { get; set; }

        [FindsBy(How = How.XPath, Using = "//android.view.View[@id='com.bungii.customer:id/toolbar']/android.widget.ImageView")]
        public IWebElement CloseRateTipPage { get; set; }

        [FindsBy(How = How.XPath, Using = "//android.widget.TextView[@text ='Rate Your Driver']")]
        public IWebElement Title_RateYourDriver { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/driver_profile_image")]
        public IWebElement Image_Driver { get; set; }

        [FindsBy(How = How.XPath, Using = "//android.widget.TextView[@instance='2']")]
        public IWebElement DriverName { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/rating_bar")]
        public IWebElement RatingBar { get; set; }

        [FindsBy(How = How.XPath, Using = "//android.widget.TextView[@text='Tip']")]
        public IWebElement Title_Tip { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/tip_value_label")]
        public IWebElement TipAmount { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/tip_increase_button")]
        public IWebElement Button_IncreaseTip { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/tip_decrease_button")]
        public IWebElement Button_DecreaseTip { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/summary_value_time")]
        public IWebElement BungiiTime { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/summary_value_distance")]
        public IWebElement Distance { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/summary_value_distance")]
        public IWebElement FinalCost { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/pickup_success_ok_button")]
        public IWebElement Button_OK { get; set; }
    }
}
